src/Common/tests/System/Drawing/Helpers.cs

```diff
@@ -12,39 +12,15 @@ namespace System.Drawing
 {
     public static class Helpers
     {
-        public const string GdiplusIsAvailable = nameof(Helpers) + "." + nameof(GetGdiplusIsAvailable);
+        public const string GdiplusIsAvailable = nameof(Helpers) + "." + nameof(IsDrawingSupported);
```

Comment:
This is still confusing, maybe this could be:

IsDrawingSupported = nameof(Helpers) + "." + nameof(GetIsDrawingSupported)

?? That way we wouldn't have any instances of GetGdiplusIsAvailable in our test code and be consistent?

Reply:
For the sake of this PR I don't want to rename 1200 callees. You can do that in a follow-up PR if you want so.

Reply:
It is just a find and replace :smile: -- but sure, I have no problem doing it afterwards.

Reply:
I'm not talking about the developer effort.

Reply:
I know, but it feels to me like 'to have a cleaner PR' I'll only make the change partially and then you can finish it up and do it in the 'correct' way. We can always split changes in multiple commits -- but as I said in my previous comment, I'm fine doing it in a follow-up PR.

Reply:
Let's go with two commits and no squash, then I'm happy ;)

Reply:
Yes, that sounds like the right plan :smile: